core/server/models/post.js

```diff
@@ -632,7 +684,7 @@ Post = ghostBookshelf.Model.extend({
      * and updating resources. We won't return the relations by default for now.
      */
     defaultRelations: function defaultRelations(methodName, options) {
-        if (['edit', 'add'].indexOf(methodName) !== -1) {
+        if (['edit', 'add', 'destroy'].indexOf(methodName) !== -1) {
```

Comment:
Needed to track related model detachments during destroy event

Reply:
makes sense 👍